newsletters/2022-05-newsletter.md

```diff
@@ -49,6 +50,17 @@ or similar:
 More details:
 [Can I use Vipps with Klarna Checkout?](https://github.com/vippsas/vipps-ecom-api/blob/master/vipps-ecom-api-faq.md#can-i-use-vipps-with-klarna-checkout)
 
+# Vipps eCom API deprecation reminder
+
+From 1. August 2022 all URL links in the body of the POST call: [/ecom/v2/payments](https://vippsas.github.io/vipps-ecom-api/#/Vipps%20eCom%20API/initiatePaymentV3UsingPOST)
```

Comment:
Hello! Here, I would have said, "From August 1, 2022, URL links in the body of the Post call [  ...  ] will no longer support HTTP, only HTTPS."